### PR TITLE
doc: add documentation framework and basic docs

### PR DESCRIPTION
**Pull request by greg-fer**
Added documentation framework files required for publishing. Added basic docs.
NCD-686.

### CODE REVIEW

File: doc/docs/overview.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+### Device
+
+The **Toggle LED** button lets you toggle the LED on the connected device.
+
+## RSSI Viewer tab
+
+This is the main area of the application where the sniffing results are displayed. Changing the side panel option settings is automatically reflected on the sniffing results when the sniffing is ongoing.
+
+![Chaning RSSI Viewer options during sniffing](./screenshots/rssi_viewer_showcase.gif "Chaning RSSI Viewer options during sniffing")
```
Comment (greg-fer) [inline doc/docs/overview.md:51]: Needs review, @kylebonnici . Please answer my questions on chat.

File: README.md [hunk 2]
```diff
@@ -3,17 +3,25 @@
 [![Build Status](https://dev.azure.com/NordicSemiconductor/Wayland/_apis/build/status/NordicSemiconductor.pc-nrfconnect-rssi?branchName=main)](https://dev.azure.com/NordicSemiconductor/Wayland/_build/latest?definitionId=9&branchName=main)
 [![License](https://img.shields.io/badge/license-Modified%20BSD%20License-blue.svg)](LICENSE)
 
-_nRF Connect RSSI Viewer_ is a tool that shows dBm per frequency in the
-2400-2480 MHz range, and allows the user to tweak settings like sweep delay and
+nRF Connect RSSI Viewer is a tool that shows dBm per frequency in the
+2400-2480 MHz range, and allows you to tweak settings like sweep delay and
```
Comment (datenreisender) [inline README.md:7]: To fix the linter issue:
```suggestion
nRF Connect RSSI Viewer is a tool that shows dBm per frequency in the 2400-2480
MHz range, and allows you to tweak settings like sweep delay and animation
duration.
```

File: doc/docs/index.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# nRF Connect RSSI Viewer
+
+nRF Connect RSSI Viewer is a cross-platform tool that shows dBm per frequency in the 2.4 GHz range. It scans the 2400-2480 MHz range, records signal level on particular frequencies, and presents combined results in several ways.
```
Comment (datenreisender) [inline doc/docs/index.md:3]: “particular frequencies” and “combined results in several ways” seems a bit wishy-washy to me. How about:

```suggestion
nRF Connect RSSI Viewer is a cross-platform tool that shows dBm per frequency in the 2.4 GHz range. It scans the 2400-2480 MHz range, records signal level on the Bluetooth Low Energy channel frequencies, and presents the results.
```

File: doc/docs/overview.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+
+| Color                 | Meaning                                                                          |
+| --------------------- | -------------------------------------------------------------------------------- |
+| Green column          | |
+| Blue column           | |
+| Dark blue             | |
+| Light blue            | |
+| Dark green            | |
+| Light green           | |
+| Numerical value       | |
```
Comment (datenreisender) [inline doc/docs/overview.md:71]: Do you need assistance with filling in the meanings here?

**Review by datenreisender — COMMENTED** [review]
For such a small app the documentation seems almost unnecessary to me.

---

AFAICS we replicated similar github workflows for zoomin in many projects. 

For the label checker we added [a generic workflow in shared](https://github.com/NordicSemiconductor/pc-nrfconnect-shared/blob/main/.github/workflows/labels.yml) and then reference that in the projects ([example](https://github.com/NordicSemiconductor/pc-nrfconnect-rssi/blob/main/.github/workflows/labels.yml)). 

How about if we do something similar for zoomin?

---

You are probably aware that `doc/docs/screenshots/rssi_viewer_showcase.gif` shows an old version of the app.

--- 

You copied `resources/screenshot.gif` to `doc/docs/screenshots/rssi_viewer_showcase.gif`. How about removing the former (and adjusting the link in `README.md`)? And also removing the unused `resources/rssi_viewer.jpg`?

File: doc/docs/overview.md (hunk 4 above)
Comment (greg-fer) [reply doc/docs/overview.md:71]: I have just gotten the info needed, just need to write it. Thanks for the offer, though!

**Review by datenreisender — APPROVED** [review]
Looks good! 👍

Unless you want to keep it for archaeological reasons, you can also remove [`resources/rssi_viewer.jpg`](https://github.com/NordicSemiconductor/pc-nrfconnect-rssi/blob/bf5f2b9f03bfc247662cce28acec3e5b0718e1e9/resources/rssi_viewer.jpg).

**Pull request merged**